src/options.ts

```diff
@@ -35,39 +34,6 @@ window.addEventListener('DOMContentLoaded', async function () {
     const port = tabConnect()
     port.postMessage({ action: 'recover' })
 
-    const settingsContainer = document.getElementById('settingsSection') as HTMLSelectElement
```

Comment:
How does the options page function with this code removed?

Reply:
I don't think option.html is needed for this extension because options are for changing extension settings.
There were no settings other than language settings and it was very simple, so I deleted option.html.

Reply:
Ok, thanks. Where would a user change their language now? In the popup?

Reply:
Yes,
The users can change the language to press setting icon of popup